### PR TITLE
docs: add homebrew installation note

### PR DESCRIPTION
**Pull request by chenrui333**
relates to https://github.com/Homebrew/homebrew-core/pull/118218

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ If you're a [**Windows Scoop**](https://scoop.sh) user, then you can install osv
 scoop install osv-scanner
 ```
 
+If you're a [Homebrew](https://brew.sh/), you can install [osv-scanner](https://formulae.brew.sh/formula/osv-scanner) via:
```
Comment (oliverchang) [inline README.md:46]:
```suggestion
If you're a [Homebrew](https://brew.sh/) user, you can install [osv-scanner](https://formulae.brew.sh/formula/osv-scanner) via:
```

**Review by oliverchang — APPROVED** [review]
Thank you! Just applied one small suggestion to fix the grammar.

**Comment by chenrui333** [comment]
> Thank you! Just applied one small suggestion to fix the grammar.

nice catch, thanks!! :)

**Comment by jwillker** [comment]
Relates https://github.com/google/osv-scanner/issues/55

**Pull request merged**